content/en/resources/publications.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Publications
+---
+
+The following papers provide detailed information on securing software updater
+systems, TUF's design, attacks on package managers, and package management
+security:
+
+- [Mercury: Bandwidth-Effective Prevention of Rollback Attacks Against Community Repositories](https://theupdateframework.io/papers/prevention-rollback-attacks-atc2017.pdf?raw=true)
```

Comment:
Make all of the links in this page paths rather than external URLs, for example:

```suggestion
- [Mercury: Bandwidth-Effective Prevention of Rollback Attacks Against Community Repositories](/papers/prevention-rollback-attacks-atc2017.pdf)
```